plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -111,6 +111,9 @@ func init() {
 			Weight: 1,
 		},
 	)
+
+	// Fit is determined by node memory pressure condition.
+	factory.RegisterFitPredicate("NodeMemoryPressure", predicates.NodeMemoryPressurePredicate)
```

Comment:
I was wondering what should be the best place to put it in and if it should be run by default. Thinking about it for the second time, it should be as the kubelet will evict all pods by default if its  node is under memory pressure. @derekwaynecarr proof me wrong if we need the predicate in the kubelet admission controller. The predicate is mainly for excluding the node from scheduling best effort pods. So, scheduler should be enough.